packages/core/src/components/TermList/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+import Term from './Term';
+
+export { Term };
+
+export type Props = {
+  children: NonNullable<React.ReactNode>;
+};
+
+function TermList({ children, cx, styles }: Props & WithStylesProps) {
+  return <dl className={cx(styles.termList)}>{children}</dl>;
+}
+
+export default withStyles(({ unit, font }) => ({
+  termList: {
+    // Reset dl styles
+    marginBlockStart: 0,
```

Comment:
Do these new rules actually compile correctly? (We support more than just chrome)